### ISSUE TITLE
dwt.py error line 254 in go "handle is invalid"

### ISSUE THREAD

**Issue by Chciken12** [10se1ucgo/DisableWinTracking#160]
**INFORMATION:**

* Windows Edition (Home):
* Architecture (64bit):
* Options used: As Administrator

**LOG:**

```python
17:49:36 INFO: Python 3.5.1 (v3.5.1:37a07cee5969, Dec  6 2015, 01:38:48) [MSC v.1900 32 bit (Intel)] on win32
17:49:37 INFO: uname_result(system='Windows', node='williams-laptop', release='10', version='10.0.10586', machine='AMD64', processor='Intel64 Family 6 Model 69 Stepping 1, GenuineIntel')
17:49:37 INFO: DisableWinTracking version 3.0
17:51:02 CRITICAL: Traceback (most recent call last):
  File "dwt.py", line 254, in go
  File "dwt_util.py", line 45, in ip_block
  File "subprocess.py", line 629, in check_output
  File "subprocess.py", line 696, in run
  File "subprocess.py", line 914, in __init__
  File "subprocess.py", line 1113, in _get_handles
OSError: [WinError 6] The handle is invalid

17:51:17 CRITICAL: Traceback (most recent call last):
  File "dwt.py", line 254, in go
  File "dwt_util.py", line 45, in ip_block
  File "subprocess.py", line 629, in check_output
  File "subprocess.py", line 696, in run
  File "subprocess.py", line 914, in __init__
  File "subprocess.py", line 1113, in _get_handles
OSError: [WinError 6] The handle is invalid

17:51:21 CRITICAL: Traceback (most recent call last):
  File "dwt.py", line 254, in go
  File "dwt_util.py", line 45, in ip_block
  File "subprocess.py", line 629, in check_output
  File "subprocess.py", line 696, in run
  File "subprocess.py", line 914, in __init__
  File "subprocess.py", line 1113, in _get_handles
OSError: [WinError 6] The handle is invalid

17:51:28 CRITICAL: Traceback (most recent call last):
  File "dwt.py", line 254, in go
  File "dwt_util.py", line 45, in ip_block
  File "subprocess.py", line 629, in check_output
  File "subprocess.py", line 696, in run
  File "subprocess.py", line 914, in __init__
  File "subprocess.py", line 1113, in _get_handles
OSError: [WinError 6] The handle is invalid

17:51:36 CRITICAL: Traceback (most recent call last):
  File "dwt.py", line 254, in go
  File "dwt_util.py", line 45, in ip_block
  File "subprocess.py", line 629, in check_output
  File "subprocess.py", line 696, in run
  File "subprocess.py", line 914, in __init__
  File "subprocess.py", line 1113, in _get_handles
OSError: [WinError 6] The handle is invalid

17:51:42 INFO: Done. It's recommended that you reboot as soon as possible for the full effect.
17:51:42 INFO: If you feel something didn't work properly, please press the 'Report an issuebutton and follow the directions

```

**Comment by 10se1ucgo** [comment]
Fixed in ada20b4. Uploading release as soon as I get my virtualenv running.

**Comment by 10se1ucgo** [comment]
https://github.com/10se1ucgo/DisableWinTracking/releases/tag/v3.0.1
